docs/changelog/92436.yaml

```diff
@@ -0,0 +1,7 @@
+pr: 92436
+summary: Add `ignore_missing_component_templates` config option
+# The bot was creating `area: "Data streams, Indices APIs"` but this seems to be invalid
```

Comment:
@dakrone I'll remove this line but maybe worth someone checks on why this is happening. It seems the bot can't handle 2 labels well (or the other way around).

Reply:
That's because PRs only support a single green label, so the entry does not get duplicated in the changelog. This change would be indices APIs more than data streams (you don't have to change the labels, since you fixed this.)